pipelines/nextflow/subworkflows/additional_seq_prepare.nf

```diff
@@ -36,25 +37,34 @@ workflow additional_seq_prepare {
         gb_file = DOWNLOAD_GENBANK(accession)
 
         // Parse data from GB file into GFF3 and json files
-        EXTRACT_FROM_GB(prefix, production_name, gb_file)
-        PROCESS_GFF3(EXTRACT_FROM_GB.out.gene_gff, EXTRACT_FROM_GB.out.genome, accession)
-        GFF3_VALIDATION(PROCESS_GFF3.out.gene_models)
+        EXTRACT_FROM_GB(DOWNLOAD_GENBANK.out.downloaded_gb_data, prefix, production_name)
+
+        PROCESS_GFF3(EXTRACT_FROM_GB.out.gene_gff, EXTRACT_FROM_GB.out.genome)
```

Comment:
```suggestion
        PROCESS_GFF3(EXTRACT_FROM_GB.out.gene_gff, EXTRACT_FROM_GB.out.genome, accession)
```
PROCESS_GFF3 expects 3 inputs ?

Reply:
The issue is changes to process_gff3.nf are not in this PR sorry....